src/CDSAPI.jl

```diff
@@ -4,6 +4,80 @@ using HTTP
 using JSON
 using Dates
 
+using ScopedValues
+
+const KEY = ScopedValue("")
+const URL = ScopedValue("https://cds.climate.copernicus.eu/api")
```

Comment:
```suggestion
const URL = ScopedValue("https://cds.climate.copernicus.eu/api")
const KEY = ScopedValue("")
```

Reply:
I just noticed, If we provide the default URL in the scoped values, it will always overwrite any other url provided in any other way. I think this is not desirable as it would lead to unexpected behaviour. overwriting through the scoped values should always be explicit in the code.
I propose we reverse this change.

Reply:
Good point. Please feel free to reverse the change 👍🏽